templates/code-header.js

```diff
@@ -0,0 +1,12 @@
+/*
+Copyright <%= YEAR %> Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
```

Comment:
@tmathern 
This looks awful without the `*` in front of every line.

Must look like this: https://git.corp.adobe.com/nui/aio-cli-plugin-asset-compute/blob/6b124d14b9cf8d7c5ad3913f7efad023631e733a/src/base-command.js#L1-L11